### ISSUE TITLE
knex@0.95.1 fresh install doesn't install typescript definitions.

### ISSUE THREAD

**Issue by immutablesyntax** [knex/knex#4343]
# Environment

Knex version: 0.95.1
Database + version: mysql@2.18.1
OS: Ubuntu 20.04

# Bug

-knex@0.95.1 fresh install through npm doesn't install typescript defintions for the package

Fix - fixed the issue by installing previous stable version of knex@0.21.19 and then updating the knexjs package. The type definitions started showing again.

**Comment by kibertoad** [comment]
Do you use TypeScript version 4.1 as per https://github.com/knex/knex/blob/master/UPGRADING.md?

**Comment by immutablesyntax** [comment]
Yes, I do. I am currently on Typescript 4.1.5. 

Another issue I found that the intellisense for knex methods also don't show up on a fresh install of knex. Only after reverting back to previous version and then upgrading to 0.95.1 they start to show up again. 

A fresh install of knex@0.95.1 through npm only adds 17 packages, while knex@0.21.19 used to add 153 packages.

**Comment by kibertoad** [comment]
@immutablesyntax We dropped liftoff, that reduced amount of dependencies.

**Comment by kibertoad** [comment]
Note that knex definitions shouldn't actually show up, as knex embeds those types now.
